v2/azure-arm.yaml

```diff
@@ -1247,13 +1247,13 @@ objectModelConfiguration:
         passwordSecretRef:
           $isSecret: false
         Identity:
-          $referenceType: arm
+          $referenceType: arm+wellknown
```

Comment:
Maybe worth a comment on these with a link to either the ASO issue and/or the documentation from the service that explains what the well known reference actually is?

Reply:
Don't necessarily need to repeat it on every one either, could just put it once and then say it applies to all of the identity references in `app`.